financialaid/views.py

```diff
@@ -5,25 +5,32 @@
 
 from django.conf import settings
 from django.contrib.auth.mixins import UserPassesTestMixin
+from django.contrib.auth.models import User
 from django.db.models import F
 from django.views.generic import ListView
 from rest_framework.authentication import SessionAuthentication
+from rest_framework.exceptions import ValidationError
 from rest_framework.generics import (
     CreateAPIView,
     get_object_or_404,
     UpdateAPIView
 )
 from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
 from rolepermissions.verifications import has_object_permission
 
 from courses.models import Program
-from ecommerce.models import CoursePrice
+from dashboard.models import ProgramEnrollment
 from financialaid.models import (
     FinancialAid,
     FinancialAidStatus,
     TierProgram
 )
-from financialaid.permissions import UserCanEditFinancialAid
+from financialaid.permissions import (
+    UserCanEditFinancialAid,
+    UserCanViewLearnerCoursePrice
```

Comment:
Since we're not putting `user_id` into the API we can probably get rid of this permission for now